tests/foreman/api/test_syncplan.py

```diff
@@ -367,10 +367,8 @@ def test_positive_update_interval(module_org, interval):
     if interval == SYNC_INTERVAL['custom']:
         sync_plan.cron_expression = gen_choice(valid_cron_expressions())
     sync_plan = sync_plan.create()
-    # get another random interval
-    new_interval = gen_choice(valid_sync_interval())
-    while new_interval == interval:
-        new_interval = gen_choice(valid_sync_interval())
+    # ensure "new interval" not equal to "interval"
+    new_interval = 'hourly' if interval != 'hourly' else 'daily'
```

Comment:
Do you mean something else here :
```suggestion
    new_interval = 'daily' if interval != 'hourly' else 'hourly'
```

Because when I read it, it sounds weird as `hourly, if the interval is not equal to hourly else daily` ! Thats total opposite to the condition :)

Reply:
no, we want to set a different interval than the one that was parametrized. So set it to 'hourly' if the parametrized `interval` is not hourly. Otherwise set it to 'daily'`.

Reply:
@jyejare we are just trying to use a simple method to ensure `new_interval` is not the same as `interval` so as not to have to call the `valid_sync_interval()` function which Mike has changed so as to display user friendly values. See https://github.com/SatelliteQE/robottelo/pull/8760#issue-686984814